src/GraphQL.php

```diff
@@ -239,7 +239,7 @@ function (array $errors, callable $formatter): array {
 
         // Allow listeners to manipulate the result after each resolved query
         $this->eventDispatcher->dispatch(
-            new ManipulateResult($result)
+            new ManipulateResult($result, $query)
```

Comment:
Just the query string without the operation name is ambigious, there might be multiple operations in there.